link/utils.py

```diff
@@ -47,17 +54,10 @@ def list_to_dataframe(rows, names):
     :params rows: the data you want to put in the dataframe
     :params names: the column names for the dataframe
     """
-    from pandas import DataFrame
-    col_convert_func = None
-    try:
-        import pandas._tseries as lib
-        col_convert_func = lib.convert_sql_column
-    except ImportError:
-        import pandas.lib as lib
-        try:
-            col_convert_func = lib.convert_sql_column
-        except:
-            col_convert_func = partial(lib.maybe_convert_objects, try_float=1)
+    if pd is None:
+        raise RuntimeError('pandas is required to use dataframes')
+
+    col_convert_func = partial(pd._libs.lib.maybe_convert_objects, try_float=1)
```

Comment:
From what I can tell, no versions of `pandas` since 2018 provide the `convert_sql_column` function (see [here](https://github.com/pandas-dev/pandas/pull/19363)).

This change uses the import path for that function for `pandas` 1.0.0 and later.